doc/source/styling.rst

```diff
@@ -0,0 +1,108 @@
+.. _styling:
+
+=================
+Customising plots
+=================
+
+.. highlight:: bash
+
+Command-line options
+====================
+
+Several of Euphonic's :ref:`cl-tools` produce 2D plots. A few command-line
+arguments are provided to tweak the plot size, font settings and colour maps, e.g.::
+
+  euphonic-dos --font Monaco --fontsize 18 --figsize 10 10 --figsize-unit cm quartz.castep_bin
+
+will produce a plot with larger (hopefully monospace) text on a small
+square canvas. This may be especially useful with the ``--save-to``
+option to create consistent plots for presentation slides and academic
+papers.
+
+The ``--font`` option will be passed to the Matplotlib library as the
+preferred "sans-serif" option, and the font family will be set to
+sans-serif. Unfortunately it can be tricky to identify exactly which
+font names are accepted by Matplotlib. We cannot advise on this for
+all platforms, so if you have a preferred font it may be worth
+searching for help on using this font with Matplotlib.
+
+Using Matplotlib styles
+=======================
+
+These and other appearance customisations can be defined as a
+Matplotlib style sheet.
+`A number of popular styles are predefined in Matplotlib <https://matplotlib.org/stable/gallery/style_sheets/style_sheets_reference.html>`_
+and may be accessed by name, e.g.::
+
+  euphonic-dos --style seaborn quartz.castep_bin
+
+will yield a plot on a grey background with white gridlines.
+
+.. image:: figures/plot-styling-seaborn.png
+  :width: 400
+  :alt: PDOS plot with thin dark blue, red and green lines against a
+        pale blue background divided by white gridlines. There are no
```

Comment:
should `blue` be `grey`?

Reply:
Yup! Initially I used a different example.